src/core/server/logging/layouts/json_layout.ts

```diff
@@ -42,6 +42,10 @@ export class JsonLayout implements Layout {
   }
 
   public format(record: LogRecord): string {
+    const spanId = record.meta?.span?.id ?? record.spanId;
```

Comment:
What's the difference between the span id we get from `meta` and the one we get directly off the record?
Are the id's in `meta` the ones that are passed along?